core/commons/che-core-commons-tracing/src/main/java/org/eclipse/che/commons/tracing/TracingUtil.java

```diff
@@ -0,0 +1,22 @@
+package org.eclipse.che.commons.tracing;
+
+import io.opentracing.Span;
+
+public final class TracingUtil {
+
+  public static void setErrorStatus(Span span, Throwable e) {
```

Comment:
What do you think about moving this method to `TracingTags` ?

Reply:
I like the idea

Reply:
Great idea, it is done (and this class is removed completely). I was still stuck in the previous PR's mode of thinking.